templates/rule-change-proposal.md

```diff
@@ -0,0 +1,19 @@
+**What rule do you want to change?**
+
+
+**Does this change cause the rule to produce more or fewer warnings?**
+
+
+**How will the change be implemented? (New option, new default behavior, etc.)?**
+
+
+**Please provide some example code that this change will affect:**
+
+```js
+<!-- example code here -->
```

Comment:
No. The html comment won't be output of its left in by mistake, a JS comment will